packages/next/build/index.ts

```diff
@@ -683,12 +683,29 @@ export default async function build(
       process.env.NEXT_PHASE = PHASE_PRODUCTION_BUILD
 
       const staticCheckWorkers = new Worker(staticCheckWorker, {
+        timeout: 60000,
```

Comment:
Would there be any valid use case where `getStaticPaths` takes more than a minute? Should we increase the timeout?

Reply:
Might be good to start warning first to see if there's cases where this is the case

Reply:
I could also make it an config option?

Reply:
Yeah maybe make it experimental initially

Reply:
Opt-in or configurable but defaulting to 1min?

Reply:
I did that:

> configurable but defaulting to 1min

and added links to the error messages which point to the config options to change it